addons/mail/static/src/models/thread/thread.js

```diff
@@ -999,6 +999,21 @@ function factory(dependencies) {
             });
         }
 
+        /**
+         * Changes description of the thread to the given new description.
+         * Only makes sense for channels.
+         *
+         * @param {string} newDescription
+         */
+        async changeDescription(newDescription) {
+            return this.env.services.rpc({
+                model: 'mail.channel',
+                method: 'channel_change_description',
+                args: [this.id],
```

Comment:
Just for consistency, I prefer if we give a list of id here

Reply:
@seb-odoo Well, it is already a list of id isn't it ?